### PR TITLE
John Wiegley 0:30 - 0:45

### PR DESCRIPTION
**Pull request by tuzz**
Here are my amendments.

This time was a bit quicker still, taking about an hour and a half in total.

There was one segment I couldn't make out so I've marked as [inaudible]: https://github.com/tuzz/captions/commit/c35447cf

### CODE REVIEW

File: 4/john-wiegley-captions.srt [hunk 1]
```diff
@@ -1744,30 +1749,31 @@ So we're going to say:
 
 399
 00:32:37,800 --> 00:32:41,600
-inaudible is not in scope, this comes from inaudible
+forM_ is not in scope,
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:1752]: Nice work!

File: 4/john-wiegley-captions.srt [hunk 2]
```diff
@@ -1827,7 +1834,7 @@ I need to pack the string.
 
 418
 00:34:19,500 --> 00:34:29,700
-inaudible
+[inaudible]
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:1837]: I hear:

```
OC: I guess parse is a pure function
JW: Yeah, we're working with pure functions here
```

File: 4/john-wiegley-captions.srt [hunk 3]
```diff
@@ -1857,7 +1864,8 @@ infinite loop (laughter).
 
 425
 00:34:56,000 --> 00:34:58,600
-inaudible tracing.
+It'd be nice to know why,
+so let's add some tracing.
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:1868]: Nice one!

File: 4/john-wiegley-captions.srt [hunk 4]
```diff
@@ -1930,17 +1939,17 @@ only being called once.
 
 442
 00:35:51,100 --> 00:35:53,800
-Our internet recursion
+Our infinite recursion
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:1942]: :+1: good catch!

File: 4/john-wiegley-captions.srt [hunk 5]
```diff
@@ -2041,15 +2051,15 @@ JW: Yes.
...
 
 469
 00:38:43,600 --> 00:38:45,920
-We definitely want to strip out  the comments.
+We definitely want to strip out the comments.
 
 470
 00:38:50,200 --> 00:38:52,200
-Let's write a function called visage.
+Let's write a function called massage.
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:2062]: Good catch!

File: 4/john-wiegley-captions.srt [hunk 6]
```diff
@@ -2123,15 +2133,16 @@ What I would like to do next
 
 487
 00:41:57,000 --> 00:41:58,160
-is do it in Rebel style.
+is do it in a REPL style.
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:2136]: You heard it first here: JW does it Rebel style! :sunglasses:

File: 4/john-wiegley-captions.srt [hunk 7]
```diff
@@ -2201,33 +2212,34 @@ Let this be:
...
 
 507
 00:43:25,600 --> 00:43:26,800
-And our test is going to ....
+And our test is going to...
 
 508
 00:43:38,400 --> 00:43:40,900
-That inaudible not actually a unit code character.
+That's not really a unicode character.
```
Comment (nelstrom) [inline 4/john-wiegley-captions.srt:2224]: :+1: good catch!

**Comment by nelstrom** [comment]
Great work @tuzz, thanks!

File: 4/john-wiegley-captions.srt (hunk 2 above)
Comment (tuzz) [inline 4/john-wiegley-captions.srt:1837]: That sounds correct. Let's go with that!

Comment (nelstrom) [inline 4/john-wiegley-captions.srt:1837]: [Done](https://github.com/fromPeerToPeer/captions/commit/ad6e7e91fbbb9b30e95384da264523557b1a6519)

**Pull request merged**